spec/brainstem/dsl/field_spec.rb

```diff
@@ -99,4 +99,31 @@ def matches?(model, helper_instance, conditional_cache)
       end
     end
   end
+
+  describe "#optioned?" do
+    context "when is not an optional field" do
+      it 'returns true' do
+        expect(field.optioned?(['expensive_title', 'expensive_title2'])).to eq true
+      end
```

Comment:
Hey @jhmoore, you're right, that's confusing.  Those values aren't actually being used in the first case because it's not optional, but I updated it to be a little clearer I think.